scripts/dockerfiles/build/Dockerfile.parsers-al2

```diff
@@ -0,0 +1,24 @@
+# Amazon Linux Tag to use for images
+ARG AL_TAG=2
+
+FROM amazon/aws-for-fluent-bit:build-deps-al${AL_TAG}
+
+# Create required directories
+RUN mkdir -p /fluent-bit/bin /fluent-bit/etc /fluent-bit/log
+
+# Accept build args for consistency (not used in AL2)
```

Comment:
nit: What do we gain by accepting the build args if they are not used? I guess if someone is used to passing args for AL2023 then switches to AL2, would it fail for them if they (or their automation) pass these args?

Reply:
Nothing to be honest, if we decided to use them we would need to ARG them in and make use